cli/common.go

```diff
@@ -108,7 +108,7 @@ func maybeBlockWithFeedback(ctx context.Context, bctx blockWithFeedbackCtx) {
 			} else {
 				newLogger.Error(
 					ctx,
-					"Failed to %s %d %v for group '%s': %v. (%s)",
+					"Failed to %s %d %v for group '%s'. %v. (%s)",
```

Comment:
move the colon behind the parantheses please. (or are you writing sentences like this?)